src/Components/CriticalCareRecording/VentilatorParametersEditor/CriticalCare__VentilatorParametersEditor.res

```diff
@@ -149,17 +149,17 @@ let saveData = (id, consultationId, state, send, updateCB) => {
 
 let ventilatorInterfaceOptions: array<Options.t> = [
   {
-    label: "Invasive (IV)",
+    label: "Invasive ventilator (IV)",
     value: "INVASIVE",
     name: "ventilator_interface",
   },
   {
-    label: "Non-Invasive (NIV)",
+    label: "Non-Invasive ventilator (NIV)",
     value: "NON_INVASIVE",
     name: "ventilator_interface",
   },
   {
-    label: "None",
+    label: "Oxygen Support",
```

Comment:
Add Oxygen Support as a Separate Option.

Handle the same in the Edit & Display Components as well